FalmouthFalcons/iteration-3
# Jim f ticket  Corrected price total on products

Description:
Corrected product total in shopping cart on page http://localhost:3000/orders/1

Number of Fixes:
 Rounded total price and total for all items in cart, added default picture for missing images

Related Ticket(s):
User can complete an order ticket

Problem to Solve:
N/A

Proposed Changes:
None

-Describe the proposed changes.
None

Expected Behavior:
Orders page http://localhost:3000/orders/1 will have styling, default pic, rounded price total, and total for items without submitted pic.

Steps to Test Solution
1) git pull origin master
2) git fetch
3) git checkout jim-f-ticket
4) run rails db:seed (if needed)
5) Go to page show products page and add item to cart or go to  http://localhost:3000/orders/1


Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
